tests/data/console/controllers/fixtures/subdir/SecondFixture.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace yiiunit\data\console\controllers\fixtures\subdir;
+
+use yii\test\Fixture;
+use yiiunit\data\console\controllers\fixtures\FixtureStorage;
+
+class SecondFixture extends Fixture
+{
+    public function load()
+    {
+        FixtureStorage::$subdirSecondFixtureData[] = 'some data set for subdir/second fixture';
```

Comment:
if thats the case, it would be nice to have this string formulated empty or with a more understandable description `'dummy string'`